index.html

```diff
@@ -1321,6 +1321,14 @@ <h2 id="BaseAudioContext">
             </p>
           </dd>
           <dt>
+            ConstantSourceNode createConstantSource()
```

Comment:
Are we going to add more `createXXX()` methods for future nodes? I thought these factory methods were essentially deprecated, although no harm in adding them - it depends on whether we want to present a complete set of factory methods along with constructors in V1.